corehq/apps/userreports/transforms/specs.py

```diff
@@ -128,3 +131,17 @@ def transform_function(values):
             return delimiter.join(translated_values_list)
 
         return transform_function
+
+
+class MarkDownTransform(Transform):
```

Comment:
nit: 
```suggestion
class MarkdownTransform(Transform):
```
(otherwise I think we'd be referencing it as `mark_down` everywhere else)

Reply:
Oops good catch. https://github.com/dimagi/commcare-hq/pull/31282/commits/25b99d35bc153e009b9ac278c404b5bd8b757a52